getting-started/routing/routes.md

```diff
@@ -20,16 +20,16 @@ end
 
 ### Defining Routes
 
-The **routes** macro accepts a **pipeline** name and a **scope**, in which all routes define within this block will make use of the pipeline and the url will be scoped.
+The **routes** macro accepts a **pipeline** name and a **scope**, in which all routes defined within this block will make use of the pipeline and the url will be scoped.
 
-Lets say you are defining a static website and you want all your URL to be displayed as `http://www.mycoolsite.com/page` you will define your routes as:
+Lets say you are defining a static website and you want the URL to be displayed as `http://www.mycoolsite.com/page`. You will define your routes as:
 
 ```crystal
 # routes(pipeline, scope)
 routes :web, "/page"
 ```
 
-The routes macro takes a last argument, a block, within the block is where you define your routes.
+The routes macro takes a last argument and a block, within the block is where you define your routes.
```

Comment:
This could probably be a period instead of a comma.

Reply:
You're welcome! Glad to help. Changes made.